public/507.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+	<head>
+		<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+		<meta name="keywords" content="fanfiction, transformative works, otw, fair use, archive" />
+		<meta name="language" content="en-US" />
+		<meta name="subject" content="fandom" />
+		<meta name="description" content="An Archive of Our Own, a project of the
+    Organization for Transformative Works" />
+		<meta name="distribution" content="GLOBAL" />
+		<meta name="classification" content="transformative works" />
+		<meta name="author" content="Organization for Transformative Works" />
+		<title>
+                 Exceeded maximum allowed downloads per minute.
```

Comment:
This will need to be updated to match whatever text we use in the `<h3>`, since this is about posting works, not downloads.